pkgs/tools/security/notation/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "notation";
+  version = "1.0.0-rc.7";
+
+  src = fetchFromGitHub {
+    owner = "notaryproject";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-EM2QunSL88Am3zgKwgI94jET3xaVfvsa4MCtMZ3ejjU=";
+  };
+
+  vendorHash = "sha256-88PCnIm7nQB8jLzrfVOyDLXWX7RZeT31n1cwvb4Qza0=";
+
+  # Disable e2e tests
+  excludedPackages = [ "./test" ];
```

Comment:
The comment shouldn't state what it is doing in this case disabling the tests, that's quite obvious but rather why they are disabled, like failing because the need an internet connection or stuff like that.
When I build locally with tests enabled they fail because of this:
```
main module (github.com/notaryproject/notation) does not contain package github.com/notaryproject/notation/test/e2e/internal/notation
```

Reply:
maybe try ;)
> this is a Go sub-module and cannot be built directly

Reply:
I would merge this once you have changed the comment, hope that's fine with you.

Reply:
Done. Thanks.

Reply:
now the last thing to do is waiting for ci :D